libs/wingsdk/src/simulator/graph.ts

```diff
@@ -1,9 +1,11 @@
+import { PolicyStatement } from "./simulator";
 import { resolveTokens } from "./tokens";
 
 export interface Definition {
   path: string;
   deps?: string[];
   props?: Record<string, any>;
+  policy?: PolicyStatement[];
```

Comment:
Why not just model policy as a prop?

What's the motivation for the royal treatment?

Reply:
Nothing special - either probably works alright. My thinking was to reserve "props" for config unique to the individual resources